.github/workflows/catalog-updates-check.yaml

```
@@ -0,0 +1,60 @@
+name: Check for unsafe catalog updates in latest-dev
+"on":
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
+jobs:
+  # Check if the PR has updated latest-dev. If a catalog update is required, but latest-dev
+  # is unchanged, this will be caught in the update tests so we don't need to double check here.
+  # Otherwise, read latest-dev contents to determine if there are column added or dropped from
+  # catalog tables. A "safe" modification should rebuild the table to ensure consistent attribute
+  # numbers across versions, instead of simply doing ALTER TABLE ... ADD/DROP COLUMN
+  check_latest_dev_correctly_updated:
+    name: Check that latest-dev.sql was properly updated by PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+      - name: Check if latest-dev was modified
+        shell: bash --norc --noprofile {0}
+        id: check_updated
+        run: |
+          file="sql/updates/latest-dev.sql"
+          updated=0
+          # diff=$(git diff --name-status origin/main | grep "${file}")
+          diff=$(git --no-pager diff --name-only HEAD $(git merge-base HEAD ${{ github.event.pull_request.base.sha }}) | grep "${file}")
+          echo "diff is ${diff}"
+          if [[ ${diff} != "" ]]; then
+            updated=1
+          fi
+          echo "updated is ${updated}"
+          echo "latestdev_updated=$(($updated))" >> $GITHUB_OUTPUT
```

I guess this should also work:

```suggestion
          echo "latestdev_updated=$updated" >> $GITHUB_OUTPUT
```